app/controllers/YourBillController.scala

```diff
@@ -85,6 +86,14 @@ object YourBillController {
     InvoicePeriod(monthNumberInTaxYear(startDate), startDate, endDate, dueDate)
   }
 
+  private def chargeBearsInterest(ass: ChargeTypeAssessment): Option[IsInterestBearingCharge] = {
+    List(ass).headOption.map { (chargeTypeAssessment: ChargeTypeAssessment) =>
```

Comment:
`List(ass).headOption` will just return `ass` - couldn't you just do this?
```scala
ass.charges.exists(_.isInterestBearingCharge)
```

Reply:
Not this, but yes. Corrected. Thank you